server/src/controllers/ChapterUser/resolver.ts

```diff
@@ -140,6 +173,19 @@ export class ChapterUserResolver {
     @Ctx() ctx: Required<ResolverCtx>,
   ): Promise<ChapterUser | null> {
     await removeUserFromEventsInChapter({ userId: ctx.user.id, chapterId });
+
+    const userChapters = ctx.user.user_chapters;
+    const chapterRole = userChapters.find(
+      ({ chapter_id }) => chapter_id === chapterId,
+    )?.chapter_role.name;
+    if (chapterRole === ChapterRoles.administrator) {
+      await updateInstanceRoleForChapterRoleChange({
+        changedChapterId: chapterId,
+        newChapterRole: ChapterRoles.member,
+        user: ctx.user,
+      });
```

Comment:
```suggestion
    await updateInstanceRoleForChapterRoleChange({
      changedChapterId: chapterId,
      newChapterRole: ChapterRoles.member,
      user: ctx.user,
    });
```
Do we need the conditional?  How about leaving that logic to `getInstanceRoleName`?

Reply:
I'm seeing it as a kind of explanation why instance role might need changing. Otherwise I think some comment would be required to make that clear. What do you think?

Reply:
Good question. A comment is probably the way to go, since it's not completely obvious why we're making this check.

Something like

> Certain chapter roles have instance roles associated with them, so we have to check and update accordingly

maybe?